docs/nextsteps.rst

```diff
@@ -0,0 +1,124 @@
+.. _next-steps:
+
+Next Steps
+==========
+
+Setting Up Continuous Integration
+---------------------------------
+
+Once you have rendered your package and set it up on `GitHub
+<https://github.com>`__ you may wish to enable Travis CI, Appveyor and Read the
+Docs (configuration for these services is included in the template, but you
+could utilise others).
+
+Travis CI
+#########
+
+You should register your package on https://travis-ci.org and modify the
+``.travis.yml`` file to your needs. The default ``.travis.yml`` file contains a
+large number of builds, against various versions of Python and Astropy and
+numpy, you should choose the ones relevant to your project. Full documentation
+of the ``.travis.yml`` file can be found on the
+`Travis CI <https://docs.travis-ci.com/user/for-beginners/>`__ website.
+
+Continuous Integration services like Travis CI, continuously test your package
+for each commit, even pull requests against your main repository will be
+automatically tested, so that you notice when something breaks. For further
+information see `here
+<https://github.com/astropy/astropy/wiki/Continuous-Integration>`__ and for
+lots of example ``.travis.yml`` build configurations see `here
+<https://github.com/astropy/astropy/wiki/travis-ci-test-status>`__. Generally
+you should aim to always have your ``master`` branch work with the latest stable
+as well as the latest development version of astropy (i.e. the astropy git
+master branch) and the same versions of python and numpy supported by astropy.
+The template ``.travis.yml`` covers those versions; in some circumstances you
+may need to limit the versions your package covers.
+
+Appveyor
+########
+
+Appveyor provides testing on the Windows platform, if you want to enable this
+you should register and enable your project at https://www.appveyor.com/. The
+appveyor build is controlled by the ``appveyor.yml`` file included in the
+template (by default).
+
+Coveralls
+#########
+
+Coveralls is a web interface to monitoring what lines of code in your project
+are executed by your test suite.
+
+If you register your package with coveralls.io, you will need to uncomment the
+coveralls line in the ``.travis.yml`` file to enable upload of your coverage
+statistics to coveralls.
+
+
+Read the Docs
+#############
+
+If you want the documentation for your project to be hosted by `Read the Docs
```

Comment:
Maybe add a sentence of motivation along the lines of "In addition to testing the code, it is often useful to build documentation continuously as the code is developed.  Read the Docs is a web site that provides exactly this service." or something like that?